tests/helper/connection.c

```diff
@@ -0,0 +1,52 @@
+/*******************************************************************************
+ *
+ * Copyright (c) 2024 GARDENA GmbH
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * and Eclipse Distribution License v1.0 which accompany this distribution.
+ *
+ * The Eclipse Public License is available at
+ *    http://www.eclipse.org/legal/epl-v20.html
+ * The Eclipse Distribution License is available at
+ *    http://www.eclipse.org/org/documents/edl-v10.php.
+ *
+ * Contributors:
+ *   Lukas Woodtli, GARDENA GmbH - Please refer to git log
+ *
+ *******************************************************************************/
+
+#include <assert.h>
+#include <stdbool.h>
+#include <string.h>
+
+#include "connection.h"
+#include "liblwm2m.h"
+
+#define SEND_BUFFER_MAX_LEN 100
```

Comment:
To me the buffer size seems very small. Max packet size when using block transfer is 1024 + header. If would rather put 2048 here.